src/utils/createMusicMetadata.ts

```diff
@@ -16,11 +18,11 @@ const music_metadata = {
 	license: '', // CC0
 	locationCreated: '', // Bueno Aires, Argentina
 	external_url: '', // https://www.npmjs.com/package/onchain-music-metadata
-	animation_url: '', // ipfs://bafybeib2hyqehlrkizobojjhl6x7krll27uffx3zqs7pw3bbg6wz2wpc4m
+	animation_url: undefined, // ipfs://bafybeib2hyqehlrkizobojjhl6x7krll27uffx3zqs7pw3bbg6wz2wpc4m
 	project: {
-		title: '', // music nft
+		title: '', // title of project
 		artwork: {
-			uri: '', // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
+			uri: undefined, // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
```

Comment:
again, uri MUST be defined as a string to meet music metadata standards

https://gist.github.com/bretth18/df8358c840fa94946ec212f753e290dd